webdriver-spec.html

```diff
@@ -507,14 +507,14 @@
 <h2>Protocol</h2>
 
 <p>WebDriver <a>remote ends</a> must provide
- an HTTP-based, RESTish wire protocol
```

Comment:
OK, I’m happy with that explanation. Do you have any reservations, @AutomatedTester?